ui/raidboss/data/06-ew/raid/p1s.txt

```diff
@@ -0,0 +1,105 @@
+### P1S: Asphodelos: The First Circle (Savage)
+#
+# -it 'Erichthonios' -oc 'Erichthonios' -p "6629:14.1"
+
+hideall "--Reset--"
+hideall "--sync--"
+
+0.0 "--Reset--" sync / 21:........:40000010:/ window 100000 jump 0
+
+# First flail set
+0.0 "--sync--" sync /Engage!/ window 0,1
+14.1 "Heavy Hand" sync / 1[56]:[^:]*:Erichthonios:6629:/
+18.2 "--sync--" sync / 1[56]:[^:]*:Erichthonios:65F5:/
+22.8 "Aetherial Shackles" sync / 1[56]:[^:]*:Erichthonios:6625:/
+31.9 "Warder's Wrath" sync / 1[56]:[^:]*:Erichthonios:662A:/
+41.7 "Chain Pain" sync / 1[56]:[^:]*:Erichthonios:662[78]:/
+57.7 "Gaoler's Flail" sync / 1[56]:[^:]*:Erichthonios:65F[6789]:/
+
+# First tank mechanic
+72.1 "Pitiless Flail of Grace/Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
```

Comment:
```suggestion
72.1 "Pitiless Flail of Grace/Pitiless Flail of Purgation" sync / 1[56]:[^:]*:Erichthonios:660[EF]:/
```

As a small style nit, can you leave the full names in the timeline and then simplify them with English `timelineReplace` sections if needed? e.g. https://github.com/quisquous/cactbot/blob/main/ui/raidboss/data/06-ew/raid/p1n.ts#L139-L150

...same for True Flare/True Holy and Powerful Fire/Powerful Light.